invenio_app_ils/facets.py

```diff
@@ -28,3 +31,70 @@ def inner(values):
         return Range(**{field: args})
 
     return inner
+
+
+def custom_exists_filter(field, missing_val):
```

Comment:
I have trouble to understand what this filter will do... Can you improve naming and explanation? Next PR is ok ;)